package.json

```diff
@@ -5,6 +5,9 @@
   "main": "index.js",
   "homepage": "https://github.com/openaustin/",
   "author": "Open Austin",
+  "scripts": {
+    "start": "gulp clean && gulp"
```

Comment:
No I mean get rid of gulp entirely. Just use the npm scripts section for everything.

The gulpfile is pretty gross to me.

Just an idea.